### ISSUE TITLE
Variable types are unknown tokens

### ISSUE THREAD

**Issue by s1mark** [segmentio/terraform-docs#131]
### Prerequisites

Put an `x` into the box that applies:

- [x ] This issue describes a bug.
- [ ] This issue describes a feature request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

variable types are unknown tokens when they are not wrapped around `" "`

#### Actual Behavior

For the following variable 
```
variable "my_var" {
  type        = string
  description = "my_var"
  default     = "my_value"
}
```
I get the following error when running `terraform-docs markdown .`

`Unknown token: 2:10 IDENT string`

#### Expected Behavior

Properly read variable type token

### Replication Case

Describe in `actual behavior` section

### Version

Actual head `ec346a0fe8f8fc97e628908aa22d12525059a636`

**Comment by getSurreal** [comment]
This issue is not only with variable definitions.  I think this is basically a terraform 0.12 issue.  vars should no longer be wrapped in quotes, but if they are not, you get this same error.

**Comment by khos2ow** [comment]
As mentioned, this issue is related to Terraform 0.12 which we're working on in #62

**Comment by khos2ow** [comment]
Closed by #113
